src/main/java/com/openshift/jenkins/plugins/pipeline/OpenShiftImageStreams.java

```diff
@@ -130,7 +134,16 @@ public void checkout(Run<?, ?> build, Launcher launcher,
             bldName = build.getDisplayName();
         if (chatty)
             listener.getLogger().println("\n\nOpenShiftImageStreams checkout called for " + bldName);
-        // nothing to actually check out in the classic SCM sense into the jenkins workspace
+
+        EnvVars env = build.getEnvironment(listener);
+        lastCommitId = getCommitId(listener, env);
+        if (lastCommitId == null) {
+            String imageStream = getImageStreamName(env);
+            String tag = getTag(env);
+            String msg = "Cannot find the last tagged image for ImageStream " + imageStream + " and tag " + tag;
```

Comment:
The message should go into MessageConstants.java as a string with `%s` that we can then use String.format on.